src/HubSpot.Client/Authentication/OAuthHubSpotAuthenticator.cs

```diff
@@ -15,20 +15,22 @@ public class OAuthHubSpotAuthenticator : DelegatingHandler
     {
         private readonly OAuthOptions _options;
 
-        public OAuthHubSpotAuthenticator(IOptions<OAuthOptions> options)
+        public OAuthHubSpotAuthenticator(IOptions<OAuthOptions> options, ITokenSelector tokenSelector)
```

Comment:
Note for reviewer: I have added another component required for OAuthHubSpotAuthenticator named ITokenSelector which I assume will have breaking changes for the users updating this. This component is my solution to using different hubspot accounts connected to the same app. In our case, this component will handle switching between EMG and Keystone accounts. 
In addition to this, the option "refresh_token" has been replaced with "refresh_tokens" which is now a dictionary for storing the multiple tokens we plan to use (e.g. dictionary pair 1 - Key: Keystone, Token: "abc123...").
If this is not super clear, reach out to me and I can explain in more detail.